style.css

```diff
@@ -0,0 +1,118 @@
+/* @import url('https://fonts.cdnfonts.com/css/poppins'); */
+
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  margin: unset;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+#logo {
+  width: 88%;
+
+  /* width: 76px; */
+  height: 20px;
+
+  /* height: 28%; */
+  margin-top: 70%;
+
+  /* margin-top: 58px; */
+  margin-left: 50%;
+
+  /* margin-left: 35px; */
+}
+
+.fa-solid {
+  width: 18px;
+  height: 12px;
+  color: #6070ff;
+  margin-right: 1.95rem;
+  margin-top: 62px;
+  margin-top: 3.55rem;
+  font-size: 18px;
+}
+
+.back-image {
+  background-image: url("images/header.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 604px;
+}
+
+.text-container {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+h1 {
+  font-family: "Poppins", sans-serif;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 40px;
+  font-style: normal;
+  line-height: 52px;
+  width: 327px;
+  height: 100px;
```

Comment:
Well done overall 💯 but please avoid hardcoded `height` and `width` values 📏 as this can easily go wrong with fixed dimensions, like here for example, this height is making the `h1` tag overflow the `p` tag 👇🏽 

![Screenshot 2023-03-31 at 08 08 36](https://user-images.githubusercontent.com/45739963/229116036-386c81ff-eaca-4ed9-8183-8a2d626126a9.png)

When I remove the height 👇🏽 

![Screenshot 2023-03-31 at 08 09 18](https://user-images.githubusercontent.com/45739963/229116153-fb85b2b2-298f-4144-8d34-caea62d6b270.png)